code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -88,6 +88,6 @@
 	if(H.dna.features["original_moth_antennae"] != null)
 		H.dna.features["moth_antennae"] = H.dna.features["original_moth_antennae"]
 
-	if(H.dna.features["original_moth_antennae"] == null && H.dna.features["moth_antennae" == "Burnt Off"])
```

Comment:
It's because it's indexing `"moth_antennae" == "Burnt Off"`.

Change it to `H.dna.features["moth_antennae"] == "Burnt Off"`, and it will work.

Reply:
I checked character menu, there is no burnt off state for moth antennas, isn't it?